src/actions/indexing/indexable-indexing-complete-action.php

```diff
@@ -31,6 +31,6 @@ public function __construct( Options_Helper $options ) {
 	 * @return void
 	 */
 	public function complete() {
-		$this->options->set( 'indexables_indexation_completed', true );
+		$this->indexable_helper->set_indexables_completed( true );
```

Comment:
ik vind complete / finalize / finish / etc. etc. een betere naam dan set_indexables_completed; je wilt niet teveel aan de buitenwereld laten zien wat er onder water nodig is.  misschien even op thesaurus.com kijken of er een synoniem is wat precies de functionele lading dekt?

Reply:
I'm struggling with this. The functionality ("functionele lading") is to set an option (`indexables_indexation_completed`) in the database, which we can use to get insights from tracking. But I wouldn't call the function `set_tracking`, or similar, because we might use the `indexables_indexation_completed` option for other purposes at a later point. `complete` / `finalize` / `finish` doesn't exactly cover the functionality it seems to me.

@andizer What is your opinion?

Reply:
I would choose for: `finish_indexing` because having only finish as method name might be confusing especially because it is in an helper. And I would name the getter: `is_indexing_finished`. Something like that.

Reply:
Thanks @andizer, I have implemented your suggestion (there is no getter for this option).